codalab/lib/worksheet_util.py

```diff
@@ -102,6 +102,16 @@ def string_to_tokens(s):
 
 ############################################################
 
+def get_worksheet_info_edit_command(raw_command_map):
+    '''
+    Return a cli-command for editing worksheet-info. Return None if raw_command_map contents are invalid.
+    Input:
+        raw_command: a map containing the info to edit, new_value and the action to perform
+    '''
+    if not raw_command_map.get('k') or not raw_command_map.get('v') or not raw_command_map.get('action') == 'worksheet-edit':
+        return None
+    return 'wedit -' + raw_command_map['k'][0] + ' "' + raw_command_map['v'] + '"'
```

Comment:
The structure of the output might be slightly clearer with
`'wedit -{k[0]} "{v}"'.format(**raw_command_map)`